### ISSUE TITLE
Design Flyer and Posters

### ISSUE THREAD

**Issue by DoESsean** [DoESLiverpool/somebody-should#1097]
We used to have an issue for designing flyers and business cards, but once the cards were done, it got closed.

This is kind of important.

We've been in the new space for a year, and, as we know, passing trade is hard to come by. This means that we need to be more active in encouraging people into the space, and one way to do that is by actually letting them know that we exist.

If we can get some good, bright, informative flyers and posters designed, I think it's worth spending some money to get them printed on proper stock and get them sent out across the city.

What we offer is genuinely great, and I think it's only the fact that people don't know about us that stops us being much busier.

Ideally, we'd have a few different designs, advertising different events:

- [ ] Maker Night/Day

- [ ] General DoES Activities and Offerings

- [ ] Jelly Tots

They would be my suggestions as a start point, feel free to add your own.

@ajlennon has spoken to Marti, who says he's happy to help but will need some collateral to build from, and is going to be short on bandwidth for a little while. The more we can offer up, the better - if we could get to a stage where images, logos, text, etc is sorted and then it's just a design job, that would be perfect.

If there is interest, it will probably be worth putting together a Hack the Marketing day/evening, where we can pool our ideas and come up with some options.

**Comment by DoESsean** [comment]
I've created some collateral for this, it's in Google Drive, so if anyone wants access to it I should be able to give it to you.

It's very simple - some images which may work, the DoES logo, some basic copy. @zarino is going to have a look (and no doubt improve it tenfold!) when he has chance to, but if anyone else wants to comment, all input is welcomed.

**Comment by ajlennon** [comment]
Thought of another place to put a MakerNight/MakerDay flyer - in the receiption at LJMU Byrom Street

**Comment by ajlennon** [comment]
To add to the notice board at Roast 92 (btw I'm happy to take all these around wherever when we have a flyer we like for makernight/makerday. Hoping we have something that isn't dated so it can stay up for a long time?)

**Comment by martimorta** [comment]
Hi, I am happy to give it a go, but I would like to have the information you would like to have in each flyer or poster, is that information available somewhere? Thanks!

**Comment by DoESsean** [comment]
Hi @martimorta, there's some collateral in the Somewhere Safe repo, [https://github.com/DoESLiverpool/somewhere-safe/issues/1](url)

**Comment by DoESsean** [comment]
Right, I have designed a poster for the Maker Events. This is heavily caveated with the facts that a) I'm not a designer, b) I have zero design skills, and c) I AM NOT A DESIGNER.

Anyway, if people who are infinitely better than I am can see even the tiniest smidgen of value in this and tinker with it/rip it apart and start again so that it is viable, I can start looking at getting some printed and spread across the city. 


[Maker Poster.docx](https://github.com/DoESLiverpool/somebody-should/files/3162714/Maker.Poster.docx)

**Comment by DoESsean** [comment]
Right, I have designed a poster for the Maker Events. This is heavily caveated with the facts that a) I'm not a designer, b) I have zero design skills, and c) I AM NOT A DESIGNER.

Anyway, if people who are infinitely better than I am can see even the tiniest smidgen of value in this and tinker with it/rip it apart and start again so that it is viable, I can start looking at getting some printed and spread across the city.

(This is a pdf. It was originally done in Libre Office, and saved as an .odt, which isn't supported by github, but saving it as a .docx destroyed all the formatting).

[Maker Poster.pdf](https://github.com/DoESLiverpool/somebody-should/files/3162738/Maker.Poster.pdf)

**Comment by jackie1050** [comment]
I think you've picked a good range of photos with different people and activities - just wondering if it would be better to use one of the other photos of Jessica (without her business name)?
________________________________

**Comment by DoESsean** [comment]
Can do, I thought that one showed the cutting quite nicely (and I'm not averse to having someone else's business name on there, to be fair, I'm not sure if it's even that obvious that's what it is).

Follow up question - as all these pictures (or whatever pictures do end up being used) have been posed for for our Instagram, and therefore it's known that they would be used for advertisement, do we need to contact the people in them to make sure they're happy to be on posters? I'd likely be inclined to anyway I think, but just in case someone doesn't reply or something.

**Comment by jackie1050** [comment]
We could easily ask them, I expect they'd say yes - that would resolve what she thinks about her name being there too.
________________________________

**Comment by johnmckerrell** [comment]
Good effort. I'd probably swap Will and Jessica so that Jessica's head doesn't get chopped off by our logo. I'm not a big fan of how the text is laid out. I thought it was because you'd used the Transport font but now I'm not so sure. I think it needs more vertical spacing and just a bit of tweaking to make sure the blocks of text don't have single words on the last line.

**Comment by seanspotatobusiness** [comment]
I made a poster using some of the same images with no attention to printer margins. The map is probably subject to copyright and not really necessary. I did an alternative cut without it.

Where are the posters going to go? Are they going to be A3 or A4? I guess the photo quality should be improved before printing at A3. I guess the flyers will be double-sided?

If pictures are being taken in low light, it would be good to find a stable place to put the camera because of some of these are a little blurry.

![DoES poster maker events](https://user-images.githubusercontent.com/4510633/57517032-d82b5b80-730d-11e9-9f8e-e13f3e38abe7.png)

![DoES poster maker events alt](https://user-images.githubusercontent.com/4510633/57518330-b089c280-7310-11e9-994f-9df606fbc179.png)

**Comment by johnmckerrell** [comment]
Nice! I like the idea of including a map, if you include an OpenStreetMap map and credit it properly that will work. Shout if you need any help with that.

**Comment by jackie1050** [comment]
I like the second one. I think people would definitely be interested.

I don't know if it's that practical for Instagram to only take pictures when you can get a stable platform / good light - my view was that it's ok to have an "informal" look - not sure if others agree - but it would be good to get some nice pictures of some of Rob's more recent stuff.
________________________________

**Comment by seanspotatobusiness** [comment]
![DoES poster maker events alt](https://user-images.githubusercontent.com/4510633/57566303-cc4da100-73c2-11e9-9b4b-3ea48b37b24f.png)

This one has an OSM map and I made a better effort to uncover people's faces.

Everyone okay with the font and colours?

**Comment by DoESsean** [comment]
We'd need to change the font I think - DoES' in house styling is Transport, which isn't a stock font.

I'm not 100% on the colours of the textboxes - I originally used the green because we've used it on one of those pull up stands before, but was relying on someone to tell me why it needed changing and what to change it to 😂 To be honest, I'm still not sure what would be better, but that's why we have proper designers in the community!

We'll definitely need to get permission of we have an image of a minor on there too, but I do really like the idea of that, showing lots of different ages.

I'm absolutely nitpicking though, thanks for improving on my original effort?

**Comment by martimorta** [comment]
Hi all, sorry for being late on this issue!
It's really cool to have the pictures of people with their results, I wonder if showing the creation process in a way would give the feeling of "you can do it all here!" sort of thing. 
I am totally up about having a map, logo and even the QR code, I think having a common footer or style could be beneficial for reusing it for the other posters.  Personally I find the style of shapes, font and colours a bit too strong though.
I would like to try and design something this weekend, @DoESsean or @johnmckerrell are you using the Transport fonts from roads.org.uk ?

Also some thoughts/ideas I am having are:
- Use a style that can be reused but at the same time make it different enough for each poster
- 2 posters for maker day and night playing with the light colours or dark colours
- the same idea but in 1 single poster
- The QR code could go to a does liverpool website where you can see a 3D model and download it to print it at Does.
- Creation process: sketch in paper, 3d model, printing/cutting, result
- Use the Does logo style (which reminds me the polaroid photos) to hold the pictures
- Another take on the posters, which would be less "human" but maybe more impactful: Several posters with different gadgets, prints, stuff, and a big slogan "MAKE IT", "DO IT", "CREATE IT"

**Comment by martimorta** [comment]
Hi, I have had a try, what do you think? I found that all pictures are more about laser cutting and woodwork, some other pictures would be handy, for 3d printing, etc. About this 2 things, one, I tried to use day pictures on the day side and night pictures on the night side, the other thing is I have added a line about the community which I think is one of the selling points of DoES, and wasn't mentioned in the collaterals.

Cheers!

![does_maker_events](https://user-images.githubusercontent.com/3284973/57586285-a243ce00-74eb-11e9-97bc-e28a768671e1.png)

**Comment by ajlennon** [comment]
Love it! It occurs to me it would be really good to have a big group photo one maker night for this?

**Comment by DoESsean** [comment]
I'll see if we have photos of people that have done sublimation printing and 3d printing for those boxes, rather than more laser cutting - it makes sense to have examples of the thing the text is referring to in the image.

I'll have to do crappy versions of things more often, it obviously gives people an 'I can improve that!' itch that they have to scratch!

**Comment by DoESsean** [comment]
@martimorta Do you have a version of that poster we would be able to edit?

Also, can we get a consensus as to which of the styles we prefer, and then we can roll out a tranche of posters with the same format for different things. That way we can finally get ourselves a presence in the wider Liverpool area.

**Comment by ajlennon** [comment]
Can we also talk about sizes and costs for printing. I'll happily tromp around putting posters up for #MakerNight and #MakerDay if we have some

**Comment by martimorta** [comment]
> @martimorta Do you have a version of that poster we would be able to edit?

yes, should I put in a Dropbox folder alongside the pictures and fonts?

**Comment by DoESsean** [comment]
> > @martimorta Do you have a version of that poster we would be able to edit?
> 
> yes, ~should I put in a Dropbox folder alongside the pictures and fonts?~
> 
> I have put it in this [Dropbox folder](https://www.dropbox.com/sh/7o29es26zjcre59/AAAk1CuCqCMYjb0CcBVpxVkaa?dl=0) if you want to edit it please, let me know your dropbox user or email since I think this is view only.

Ah, now there's an issue - the Dropbox account I use is tied to an email address that no longer exists, and when I've tried to change that, it tells me that my password (which is saved, and has managed to make sure that I'm still logged in on my computer and phone) is incorrect. I have no idea.

**Comment by amcewen** [comment]
In general it'd be better for us to be migrating all of these sort of things to the [somewhere-safe](https://github.com/doesliverpool/somewhere-safe) repositor.

**Comment by martimorta** [comment]
It makes sense, can you add me as a contributor @amcewen ?

**Comment by amcewen** [comment]
How did you manage to sneak by so long without getting invited to the @DoESLiverpool organisation?  :-D  I've just sent you an invite, which should then give you access to the repos.

**Comment by martimorta** [comment]
There you go!
 https://github.com/DoESLiverpool/somewhere-safe/tree/master/Promo/Posters/Does%20Maker%20Events

**Comment by ajlennon** [comment]
Had a chat with Max upstairs from NoFaceTattoo. He's got a plan for some flyers for marketing and I suggested we could do some co-marketing, we take his flyers around town and he takes ours. Maybe there are others in the space who would be interested in sharing the journey?

**Comment by amcewen** [comment]
The A5 flyer that @chris-does did for #1289 is now stashed https://github.com/DoESLiverpool/somewhere-safe/tree/master/Promo/Posters/A5%20General%20Flyer and we've a stack of them printed.

There's discussion of where to put some in #591, but @ajlennon feel free to liaise with Max and give him some to put out

**Comment by ajlennon** [comment]
Will do! Will take some up. Will also take some over to AYCH and see if they can help leave some around the Baltic for us

**Comment by johnmckerrell** [comment]
Hearing about the great Makefest flyer that had events listed on made me realise that if we had enough events that we could do this as a regular thing, it would be a reason to put more flyers out and might spark more interest if people saw dates against things rather than just "This is an (amazing) workshop and coworking space".

**Comment by amcewen** [comment]
From today's community meeting:

> If we have a QR code, we could have it point to a landing page (or something with simple tracking at least) so we can get an idea of how many people see it (and act on it)

**Comment by omarmorshedy** [comment]
I have created a python script that generates a flyer for DoES. It is mostly inspired by the doesliverpool website, using the same colours. This is more of a general design and is open for feedback, the fonts and images are easily interchangeable to allow for iteration, currently it only uses the images from the website. I've uploaded the script, assets it uses and some of the outputs(versioned) to the somewhere-safe repo under [flyers](https://github.com/DoESLiverpool/somewhere-safe/tree/master/Promo/Flyers)

**Comment by amcewen** [comment]
I made some tweaks to @omarmorshedy's flyer, mostly to the copy and layout.
 
* Fonts all changed to Transport New
* Yellow surround removed from logo at the top, and logo made a bit bigger
* White logo removed from lower down
* Extra "make - work - meet" text removed
* Numbers in front of tool descriptions removed
* QR code updated to one with ?utm=flyer tracking
* Couple of tweaks to copy

It still needs a new photo for the 3D printers (so they're not on their side :-D) I've [pushed it to a new branch](https://github.com/DoESLiverpool/somewhere-safe/tree/makefest-2026-version) (mostly because @omarmorshedy also has some tweaks and I didn't want to complicate them)

![updated version of the flyer](https://github.com/DoESLiverpool/somewhere-safe/blob/makefest-2026-version/Promo/Flyers/python-flyers/does-liverpool-flyer.png?raw=true)

It would be good to have more about the co-working side of things, but I think we should get a load of these (I just had a chat with @omarmorshedy and we don't think his tweaks should delay it) ordered for MakeFest and then do an updated or double-sided version that covers co-working (and events room?) in the future

**Comment by amcewen** [comment]
I've ordered 1000 of those from https://solopress.com.  Went with these options:

<img width="228" height="405" alt="Image" src="https://github.com/user-attachments/assets/c499f247-44e2-45c2-bef4-8b4649fde8e4" />

It complained that the images weren't very high resolution, so for the next iteration it would be good to try to improve that.  @willcrobinson and I were wondering about whether we could recreate the flyer in html/css and use `weasyprint`, [like we've been looking at for health-and-safety signs](https://github.com/DoESLiverpool/somebody-should/issues/243#issuecomment-3708412856)

**Comment by amcewen** [comment]
Allegedly the flyers will be delivered tomorrow (as I ordered before 12pm)

**Comment by amcewen** [comment]
The flyers *were* delivered.  They're on @DoESsean's desk

<img width="1600" height="900" alt="Image" src="https://github.com/user-attachments/assets/76d09b80-e809-4b7a-9e02-fd7efdca7fa5" />
